addons/miscFixes/fauxLAMG/CfgWeapons.hpp

```diff
@@ -0,0 +1,32 @@
+class CfgWeapons {
+    class LMG_Mk200_F;
+    class GVAR(LAMG): LMG_Mk200_F {
```

Comment:
I'm not sure how far you want to take the faux LAMG, but if you'd like to make more changes to things like weight, ROF, sound, etc. Let me know and I can lend a hand.

Reply:
Yeah! I think the only thing that I would like to change would be the sound since the vanilla game/ACE's Mk200 is pretty damn accurate to the IRL LAMG/AMG already. I always appreciate the help!